src/bud/mode.ts

```diff
@@ -0,0 +1,46 @@
+import {argv} from 'yargs'
+import {env} from './state/env'
+
+/**
+ * Fallback env
+ */
+const envFallback: string = 'production'
+
+/**
+ * specified via CLI arg
+ */
+const envArgument: any = argv.env
+
+/**
+ * specified via project .env
+ */
+const envProject: string = env?.APP_ENV || envFallback
+
+/**
+ * ## bud.mode
+ *
+ * Webpack mode ('development'|'production')
+ *
+ * Determined by the first match, in order of precedence:
+ *
+ *  - CLI args
+ *  - env file
+ *
+ * Fallback is 'production'.
+ */
+const mode: Mode = envArgument ? envArgument : envProject
+
+/**
+ * inProduction
+ */
+const inProduction: Production = mode == 'production'
```

Comment:
It might just be my own coding style, but is there any benefit in using loose equality here? I recognize that there really isn't any benefit in using strict equality also, I just tend to err on the side of only reaching for loose equality when I need it and using strict equality for most other things.

Reply:
this is now a strict check. thanks!